airbyte-integrations/connectors/source-google-ads/Dockerfile

```diff
@@ -13,5 +13,5 @@ COPY main.py ./
 
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.14
+LABEL io.airbyte.version=0.2.16
```

Comment:
Are we  skipping the version `0.2.15` here?

Reply:
Oh, I see https://github.com/airbytehq/airbyte/pull/25987/files#diff-c6131fa38258907725f3d89884807ffdb5eceb509663785bd9b32118513e193fL9 makes `0.2.15`